packages/webdriver/webdriver.d.ts

```diff
@@ -327,7 +327,7 @@ declare namespace WebDriver {
         // Testingbot w3c specific
         'tb:options'?: TestingbotCapabilities
         // Saucelabs w3c specific
-        'sauce:options'?: SauceLabsCapabilities
+        'sauce:options'?: SauceLabsCapabilities & SauceLabsVisualCapabilities
```

Comment:
As noted above, Visual options live in a new top-level capability `sauce:visual`, parallel to sauce:options